projects/objects/backgrounds/protos/TexturedBackground.proto

```diff
@@ -38,7 +38,7 @@ PROTO TexturedBackground [
       texture == "noon_cloudy_countryside" or
       texture == "noon_park_empty" or
       texture == "mountains"
-    local extension = isHDR and "hdr" or "jpg"
+    local extension = isHDR and "png" or "jpg"
     local path = "textures/cubic"
   }%
   Background {
```

Comment:
Now that we do support reflections without visual background it would be interesting to add a new field to specify if the *Url fields should be set or not (but if you prefeer I can do this in another PR).

Reply:
This is a good idea. But as you tell, this is a new feature that I prefer to implement in another PR.